README.md

```diff
@@ -775,10 +775,12 @@ If there is extra data that is required for the computation to run, it must be i
 Before you can work with the Tozny Identity service, you need to [sign up for a Tozny Platform account](https://dashboard.tozny.com/register). Create a new identity realm and register a new client application with the realm.
 
 ```js
-const realmName = '...'
-const appName = '...'
+const realmName = 'NameOfRealmGoesHere'
+// Currently 'account' is the only app name supported value
+const appName = 'account'
 // This is a URL in your application which handle password reset flows.
-const brokerTargetURL = '...'
+// The default value using the Tozny Hosted Broker is https://<TozIdBaseURL>/<YourRealmName>/recover
+const brokerTargetURL = 'https://id.tozny.com/NameOfRealmGoesHere/recover'
```

Comment:
Suggestion: using {realmName} instead of NameOfRealmGoesHere would make it more obvious to use the realm name here

Reply:
@tozny-shakeeb Updated!

Reply:
@tozny-shakeeb or did you mean to use an example realm name, not literally "realmName"?